workers/unity/Packages/com.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -100,11 +105,22 @@ private static void Generate()
                 {
                     var exitCode = RedirectedProcess.Command(Common.DotNetBinary)
                         .WithArgs(ConstructArgs(projectPath, schemaCompilerPath, workerJsonPath))
+                        .RedirectOutputOptions(OutputRedirectBehaviour.None)
+                        .AddErrorProcessing(Debug.LogError)
+                        .AddOutputProcessing(ProcessStdOut)
                         .Run();
 
                     if (exitCode != 0)
                     {
-                        Debug.LogError("Failed to generate code.");
+                        if (!Application.isBatchMode)
```

Comment:
Would it still be useful to output the above line to the log as well as the extra output from below, just to sum up?

Reply:
The application itself already prints that it failed, the double log was rather weird